cabal-install/Distribution/Solver/Modular/Explore.hs

```diff
@@ -67,8 +67,6 @@ backjump (EnableBackjumping enableBj) var initial xs =
 
```

Comment:
I just realized that the line above updates the conflict map even when `countConflicts` is false.  It should probably be something like `if countConflicts then (cs, updateCM initial cm) else (cs, cm)` to avoid creating a thunk from the conflict map when it isn't used.